### PR TITLE
New lab! -- 300 lambda cross account bucket policy

### PR DESCRIPTION
**Pull request by setheliot**
*Issue #, if available:*

*Description of changes:*
This lab is based on one I created for the JP Security Day.
It shows how to use resource-based policies for cross-account access.  It is complimentary to [300_Lambda_Cross_Account_IAM_Role_Assumption](https://github.com/awslabs/aws-well-architected-labs/tree/master/Security/300_Lambda_Cross_Account_IAM_Role_Assumption)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Security/300_Lambda_Cross_Account_Bucket_Policy/Documentation/CreateNewS3Bucket.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Create New S3 Bucket
+
+These steps will guide you to create a bucket containing some objects.
+
+1. Go to te S3 console at <https://console.aws.amazon.com/s3>
```
Comment (benjipotter) [inline Security/300_Lambda_Cross_Account_Bucket_Policy/Documentation/CreateNewS3Bucket.md]: te = the

File: Security/300_Lambda_Cross_Account_Bucket_Policy/Documentation/CreateNewS3Bucket.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+1. Click **Create bucket**
+1. For **Bucket name** supply a name. This must be unique across all buckets across AWS
```
Comment (benjipotter) [inline Security/300_Lambda_Cross_Account_Bucket_Policy/Documentation/CreateNewS3Bucket.md]: buckets in AWS

File: Security/300_Lambda_Cross_Account_Bucket_Policy/Documentation/CreateNewS3Bucket.md (hunk 1 above)
Comment (setheliot) [reply Security/300_Lambda_Cross_Account_Bucket_Policy/Documentation/CreateNewS3Bucket.md]: thanks, will fix

File: Security/300_Lambda_Cross_Account_Bucket_Policy/Documentation/CreateNewS3Bucket.md (hunk 2 above)
Comment (setheliot) [reply Security/300_Lambda_Cross_Account_Bucket_Policy/Documentation/CreateNewS3Bucket.md]: will update

**Comment by setheliot** [comment]
Thanks @benjipotter 

1. typos fixed in Commit: e194da2dfb5f25b2cc6ea0f7d2137afbe5f1bbf0
2. Also added website menu entry
3. html gen test found one error. Also fixed in same commit.  Website output attached 
image

**Pull request merged**